ui/src/components/Player/DetectionCategoryButton.tsx

```diff
@@ -17,7 +19,12 @@ export default function DetectionCategoryButton({
         alignItems: "center",
       }}
     >
-      <Image src={icon.src} alt={`${title} icon`} width={100} height={100} />
+      <Image
+        src={icon.src}
+        alt={`${title} icon`}
+        width={isDesktop ? 100 : 20}
+        height={isDesktop ? 100 : 20}
```

Comment:
The [new Next.js `Image`](https://nextjs.org/docs/pages/api-reference/components/image#responsive-images) component has some responsive support but I had some difficulties making it work with the `fill` setting, so I reverted to the legacy `Image` component with media queries. The icon and the category text were on top of each other, and without `fill` you need to specify the width/height anyway

Reply:
What do you think about having multiple copies of each image and hiding/unhiding with `display: none` on different breakpoints?

I've been trying to stay away from `useMediaQuery` because it's JS based which has [some downsides](https://github.com/orcasound/orcasite/pull/379#discussion_r1542187449) (biggest problem is layout shifts). In this case (inside a modal) it's not really an issue but don't want to encourage the pattern. Unless there's a better solution I don't know of?

Reply:
And yeah legacy `Image` isn't ideal, but we are using it elsewhere. It may get removed soon since it's been deprecated, so we'll be forced to figure it out at that point… but I guess it buys us some time